exp/addons/api/v1beta1/clusterresourcesetbinding_types.go

```diff
@@ -129,6 +129,10 @@ type ClusterResourceSetBindingSpec struct {
 	// Bindings is a list of ClusterResourceSets and their resources.
 	// +optional
 	Bindings []*ResourceSetBinding `json:"bindings,omitempty"`
+
+	// ClusterName use the cluster name to determine which cluster the resource binding applies to.
+	// +optional
+	ClusterName string `json:"clusterName,omitempty"`
```

Comment:
```suggestion
	// ClusterName is the name of the Cluster this binding applies to.
	// +optional
	ClusterName string `json:"clusterName,omitempty"`
```
I would considering also opening an issue making this mandatory at the next API bump, @killianmuldoon @sbueringer opinions?

Reply:
What about adding a check in the validation webhook preventing this value to be changed/removed once it is set?